partner_credit_limit/models/account_invoice.py

```diff
@@ -19,17 +19,14 @@ def check_limit_credit(self):
             return True
         for invoice in self:
             allowed_sale = invoice.partner_id.with_context(
```

Comment:
This logic is duplicated in the method `_check_credit_limit`, which is called just below. I think this should be removed and used instead the returning value of that method

Reply:
@luisg123v 

maybe just getting ride off  the 
```python
if self.allowed_sale:
    return False
```
at _check_credit_limit.

Reply:
That's also possible. However what I really don't like of that (and the current code) is the unnecessary use of `with_context` twice with the same parameters

Reply:
Those parameters are being used on the `compute` method of the fields `credit_overloaded` and `overdue_credit`

Reply:
Yes, that's why I think only a call to `check_credit_limit` is required, no need to compute it again here.

Reply:
I get ride of
```python
if self.allowed_sale:
    return False
```
on both methods (`sale` and `account_invoice`)